modules/serverless-logic/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,193 @@
+= Building and deploying a {context} application on Kubenetes using the {product_name} Serverless Operator
+:compat-mode!:
+// Metadata:
+:description: Build and deploy using the Kogito Serverless Workflow Operator a serverless workflow application
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, openshift
+// links
+:kogito_serverless_operator_url: https://github.com/kiegroup/kogito-serverless-operator/
+:kogito_greeting_example_url: https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-examples/serverless-workflow-greeting-quarkus
+:kaniko_issue_url: https://github.com/GoogleContainerTools/kaniko/issues/2201
+
+This document describes how to build and deploy your workflow application using a Kubernetes cluster with the link:{kogito_serverless_operator_url}[{product_name} Serverless Operator]. If you don't have any cluster operational, link:{minikube_url}[Minikube] commands are given throughout the guide.
+
+Using the {product_name} Serverless Operator, you will be able to build and deploy a {product_name} {context} application only by having a workflow definition.
+
+If you already have a container built and pushed to a container registry and you want to deploy it on the Kubernetes cluster, then you can do it without the operator following the guide xref:cloud/deploying-on-kubernetes.adoc[Deploying your {context} application on Kubernetes].
+
+The link:{kogito_serverless_operator_url}[{product_name} Serverless Operator] is currently in Alpha version, is under active development and is at the moment supporting {context} definitions that are using:
+
+* Functions
+* States
+    - Switch including dataConditions
+    - Inject including data with a transition
+* Operations including Actions containing functionRef with arguments
+* KeepActive
+* AutoRetries
+* ExpressionsLang (jq or jsonpath)
+
+.Prerequisites
+* A workflow definition.
+* A Kubernetes cluster with admin privileges 
++If you haven't got one prepare, you can use a local link:{minikube_url}[Minikube] instance.
+* `kubectl` command-line tool is installed. 
++Otherwise, Minikube provides it.
```

Comment:
```suggestion
* A Kubernetes cluster with admin privileges. If you haven't got one prepare, you can use a local link:{minikube_url}[Minikube] instance.
* `kubectl` command-line tool is installed. Otherwise, Minikube provides it.
```